org.eclipse.jgit/src/org/eclipse/jgit/lib/CommitBuilder.java

```diff
@@ -86,6 +88,8 @@
 
 	private PersonIdent committer;
 
+	private byte[] gpgSig;
```

Comment:
Don't use abbreviations. Please use the full names. Also, I'm not sure byte[] is a proper value. It probably warrants it's own object.